src/features/contact/shared.ts

```diff
@@ -32,10 +32,18 @@ export const enrichOnderwerpObjectenWithZaaknummers = (
   objecten: OnderwerpObjectPostModel[],
 ) =>
   Promise.all(
-    objecten.map(({ onderwerpobjectidentificator: { objectId } }) =>
-      fetchZaakIdentificatieByUrlOrId(systeemId, objectId),
-    ),
-  );
+    objecten
+      .filter(({ onderwerpobjectidentificator }) => {
+        // Check if this is a zaak-type object
+        return (
+          onderwerpobjectidentificator.codeObjecttype === "zgw-Zaak" ||
```

Comment:
Dit moet je ook even documenteren.

Reply:
Is dit ook afgestemd? Is het inderdaad voldoende als alleen het codeObjecttype of alleen het codeRegister matcht?